app/src/main/java/com/teambeme/beme/mypage/viewmodel/MyPageViewModel.kt

```diff
@@ -38,8 +38,29 @@ class MyPageViewModel @Inject constructor(
     val scrapScrollUp: LiveData<Unit>
         get() = _scrapScrollUp
 
-    private var page = 1
-    private var scrapPage = 1
+    private var _page: Int = 2
+    val page: Int
+        get() = _page
```

Comment:
```suggestion
    var page: Int = 2
        private set
```

primitive value는 저렇게 backing을 안해도 setter만 private로 걸어버리면 동일한 효과를 나타내더라고. 이렇게 바꾸면 어떤 이점이 있을까? 같이 생각해볼래?

Reply:
아 그러네! 새로운거 알게됐다. 고마워유
그리고 생각해보니 page는 뷰모델 밖에서 참조하지도 않는 변수라 그냥 `private var page: Int =2 ` 로만 해도 될 것같은데 그게 낫겠지?

Reply:
오 좋습니다!